docs/middleware.rst

```diff
@@ -383,7 +383,7 @@ retrieved using JSON-RPC endpoints that don't rely on server state.
     >>> from web3 import Web3, EthereumTesterProvider
     >>> w3 = Web3(EthereumTesterProvider)
```

Comment:
I think EthereumTesterProvider should have parens too? 

`w3 = Web3(EthereumTesterProvider())`

Reply:
Doh! Thanks.